docs/docs/configuration/mapper.mdx

```diff
@@ -136,6 +136,41 @@ public class CarDto
 }
 ```
 
+### `null` values
```

Comment:
IMO this section is too confusing even for advanced users. As far as I can tell, `ThrowOnPropertyMappingNullMismatch` takes precedence over `AllowNullPropertyAssignment`, correct? If so, that should be enough documentation and you could remove a huge part of this section.

Reply:
I tried to improve the documentation. It is not true, that `ThrowOnPropertyMappingNullMismatch ` takes precedence over `AllowNullPropertyAssignment` or vice versa.